dvc/utils/serialize/__init__.py

```diff
@@ -1,2 +1,8 @@
+from collections import defaultdict
+
+from ._common import *  # noqa, pylint: disable=wildcard-import
 from ._toml import *  # noqa, pylint: disable=wildcard-import
 from ._yaml import *  # noqa, pylint: disable=wildcard-import
+
+PARSERS = defaultdict(lambda: parse_yaml)  # noqa: F405
+PARSERS.update({".toml": parse_toml})  # noqa: F405
```

Comment:
Also, it'd be good to have JSON utils after the upcoming PR with trees are done. Maybe even move to `ujson` as it has now wheels on all platforms.

```
In [1]: %timeit json.loads('{"ja": "son"}')
2.32 µs ± 35 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)

In [2]: %timeit YAML(typ='safe').load('{"ja": "son"}')
180 µs ± 1.28 µs per loop (mean ± std. dev. of 7 runs, 10000 loops each)

In [3]: %timeit ujson.loads('{"ja": "son"}')
372 ns ± 13.4 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```